mllib/src/main/scala/org/apache/spark/ml/param/params.scala

```diff
@@ -501,7 +501,13 @@ trait Params extends Identifiable with Serializable {
    */
   final def getDefault[T](param: Param[T]): Option[T] = {
     shouldOwn(param)
-    defaultParamMap.get(param)
+    try {
+      defaultParamMap.get(param)
+    } catch {
+      case e: NoSuchElementException =>
```

Comment:
Can you check if the map `contains` the param rather than wait for an exception? Really doesn't matter though. This looks OK except that you'll need a space after "param" in the error message below, and you could use string interpolation.